apps/crn-frontend/src/analytics/collaboration/__tests__/TeamCollaboration.test.tsx

```diff
@@ -9,6 +13,13 @@ import { getTeamCollaboration } from '../api';
 import { analyticsTeamCollaborationState } from '../state';
 import TeamCollaboration from '../TeamCollaboration';
 
+jest.mock('@asap-hub/algolia', () => ({
+  ...jest.requireActual('@asap-hub/algolia'),
+  algoliaSearchClientFactory: jest
+    .fn()
+    .mockReturnValue({} as AlgoliaSearchClient<'crn'>),
```

Comment:
I think it doesn't change anything in the test, but I think this should be
```suggestion
    .mockReturnValue({} as AlgoliaSearchClient<'analytics'>),
```